src/QMCWaveFunctions/LCAO/LCAOrbitalSetT.h

```diff
@@ -0,0 +1,336 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2016 Jeongnim Kim and QMCPACK developers.
+//
+// File developed by:
+//
+// File created by: Jeongnim Kim, jeongnim.kim@intel.com, Intel Corp.
+//////////////////////////////////////////////////////////////////////////////////////
+
+
+#ifndef QMCPLUSPLUS_SOA_LINEARCOMIBINATIONORBITALSET_TEMP_H
+#define QMCPLUSPLUS_SOA_LINEARCOMIBINATIONORBITALSET_TEMP_H
+
+#include <memory>
+#include "QMCWaveFunctions/SPOSetT.h"
+#include "QMCWaveFunctions/BasisSetBase.h"
+
+#include "Numerics/MatrixOperators.h"
+#include "Numerics/DeterminantOperators.h"
+
+namespace qmcplusplus
+{
+/** class to handle linear combinations of basis orbitals used to evaluate the Dirac determinants.
+   *
+   * SoA verson of LCOrtbitalSet
+   * Localized basis set is always real 
+   */
+template<class T>  
+class LCAOrbitalSetT : public SPOSetT<T>
+{
+public:
+  using basis_type = SoaBasisSetBase<T>;
```

Comment:
Naming convention is caps for type i.e.
```
basis_type -> Basis
vgl_type -> VGL
....
IndexType -> Index
RealType -> Real
```
etc.

This is a change to have this more consistent and easier to read I think it should be taken.